packages/shared-ui/src/components/canvas/context/FlowGraphContext.tsx

```
@@ -551,110 +562,56 @@ export function FlowGraphProvider({ children }: IFlowGraphProviderProps): ReactE
-					// Rebuild edges from the cleaned node data
-					const computedEdges = getEdgesFromNodes(updatedNodes as INode[]);
-					setEdges(computedEdges);
-
-					return updatedNodes;
-				});
-			} else {
-				// Non-removal changes (select, etc.) go straight through
-				onEdgesChangeInternal(changes);
+			const removes = changes.filter((c) => c.type === 'remove');
+			if (removes.length > 0) {
 			}
```

ESLint

[error] Empty block statement.

(no-empty)